Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/README.md

```diff
@@ -0,0 +1,181 @@
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### bitsight-get-company-details
+***
+BitSight command - to get comany details based on guid.
+
+
+#### Base Command
+
+`bitsight-get-company-details`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| guid | GUID of the company to fetch its details. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| BitSight.Company.errorCode | string | Error code number when API fails | 
+| BitSight.Company.errorMessage | string | Error Message when API fails | 
+| BitSight.Company.guid | string | guid | 
+| BitSight.Company.customId | string | customId | 
+| BitSight.Company.name | string | name | 
+| BitSight.Company.description | string | description | 
+| BitSight.Company.ipv4Count | string | ipv4Count | 
+| BitSight.Company.peopleCount | string | peopleCount | 
+| BitSight.Company.shortName | string | shortName | 
+| BitSight.Company.industry | string | industry | 
+| BitSight.Company.industrySlug | string | industrySlug | 
+| BitSight.Company.subIndustry | string | subIndustry | 
+| BitSight.Company.subIndustrySlug | string | subIndustrySlug | 
+| BitSight.Company.homePage | string | homePage | 
+| BitSight.Company.primaryDomain | string | primaryDomain | 
+| BitSight.Company.type | string | type | 
+| BitSight.Company.displayURL | string | displayURL | 
+| BitSight.Company.ratingDetails | string | ratingDetails | 
+| BitSight.Company.ratings | string | ratings | 
+| BitSight.Company.searchCount | string | searchCount | 
+| BitSight.Company.subscriptionType | string | subscriptionType | 
+| BitSight.Company.sparkline | string | sparkline | 
+| BitSight.Company.subscriptionTypeKey | string | subscriptionTypeKey | 
+| BitSight.Company.subscriptionEndDate | string | subscriptionEndDate | 
+| BitSight.Company.bulkEmailSenderStatus | string | bulkEmailSenderStatus | 
+| BitSight.Company.serviceProvider | string | serviceProvider | 
+| BitSight.Company.customerMonitoringCount | string | customerMonitoringCount | 
+| BitSight.Company.availableUpgradeTypes | string | availableUpgradeTypes | 
+| BitSight.Company.hasCompanyTree | string | hasCompanyTree | 
+| BitSight.Company.hasPreferredContact | string | hasPreferredContact | 
+| BitSight.Company.isBundle | string | isBundle | 
+| BitSight.Company.ratingIndustryMedian | string | ratingIndustryMedian | 
+| BitSight.Company.primaryCompany | string | primaryCompany | 
+| BitSight.Company.permissions | string | permissions | 
+| BitSight.Company.isPrimary | string | isPrimary | 
+| BitSight.Company.securityGrade | string | securityGrade | 
+| BitSight.Company.inSpmPortfolio | string | inSpmPortfolio | 
+| BitSight.Company.isMycompMysubsBundle | string | isMycompMysubsBundle | 
+| BitSight.Company.companyFeatures | string | companyFeatures | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
```

Comment:
You are missing the human readable outputs in all commands. you can easily add it using the `command examples` txt file and re-generate : https://xsoar.pan.dev/docs/documentation/readme_file#command-examples---integration-documentation

Reply:
When I try to run generate-docs , Got below error. So added Human Readable manually.
The provided example for cmd !bitsight-get-companies-guid has failed...